### ISSUE TITLE
autocompletion results disappearing inside :CocConfig

### ISSUE THREAD

**Issue by yujinyuz** [neoclide/coc.nvim#1741]
## versions

vim version: NVIM v0.4.3
node version: v12.16.1
coc.nvim version: 0.0.78-fc983ce6dd
term: tmux-256color
platform: darwin

## Output channel: snippets

[Info 11:32:52 PM] Using ultisnips directories: UltiSnips /Users/me/.config/coc/ultisnips
[Info 11:32:52 PM] Using ultisnips python command: pyx

## Describe the bug

When typing, the results appear but disappear after a split second

## Reproduce the bug

**We will close your issue when you don't provide minimal vimrc and we can't
reproduce it**

- Create file `mini.vim` with：

  ```vim
  set nocompatible
  set runtimepath^=/path/to/coc.nvim
  filetype plugin indent on
  syntax on
  set hidden
  ```

- Start (neo)vim with command: `vim -u mini.vim`

- Operate vim.

## Screenshots (optional)

It's appearing but disappears all of a sudden.

[![asciicast](https://asciinema.org/a/l7Cuf9kEHDtzJQdLHmIHI2mdE.svg)](https://asciinema.org/a/l7Cuf9kEHDtzJQdLHmIHI2mdE)

Here is the previous release that was working:
[![asciicast](https://asciinema.org/a/lyEMJBM01uWx0buxqlco481nk.svg)](https://asciinema.org/a/lyEMJBM01uWx0buxqlco481nk)

**Comment by voldikss** [comment]
Can confirm this issue

**Comment by voldikss** [comment]
Seems caused by af2b7f5

**Comment by iamcco** [comment]
Also happend when writing typescript:

Can not get autocomplete with below code: (`|` is cursor)

``` typescript
const hello = `hello world`

console.log(`${he|}`);
```

**Comment by mnrce** [comment]
> Also happend when writing typescript:
> 
> Can not get autocomplete with below code: (`|` is cursor)
> 
> ```ts
> const hello = `hello world`
> 
> console.log(he|);
> ```

I am at https://github.com/neoclide/coc.nvim/commit/64b63ed1e6c6517ce45b136eb4b0ef0f4ba1b10f and still can reproduce this.
I cannot trigger completion (auto trigger or even press <kbd>ctrl-space</kbd>) when I'm inside a function call.
Completion works fine when I'm **not** inside a function call.

I also tested using  `coc-go` with `go.trace.server` set to `verbose`. When inside a function call, I trigger the completion manually by pressing <kbd>ctrl-space</kbd>, I can see the completion candidates returned by `gopls` in `:CocInfo`, but there is no completion popup.

**Comment by chemzqm** [comment]
No idea, checkout https://github.com/neoclide/coc.nvim/wiki/Debug-coc.nvim

**Comment by mnrce** [comment]
I just did a `git bisect` and saw that the bug is caused by af2b7f5 as mentioned ealier by @voldikss.

I also tested ceb9e47 (the commit that closed this issue) but it didn't fix the bug for me.

I'm not familiar with typescript and coc.nvim codebase so I don't think I can help debug this issue.

Can you take a look at af2b7f5 again @chemzqm ?

**Comment by chemzqm** [comment]
I can't reproduce, please use latest release branch of coc.nvim.

**Comment by mnrce** [comment]
Sorry, I tried again with all coc extensions disabled except for coc-tsserver, and the issue goes away.
Look like the culprit is coc-postfix, there was something in ceb9e47 that caused it to break. Can you take a look at it @fannheyward?

**Comment by fannheyward** [comment]
@mnrce just released `coc-postfix` 0.0.8 with a quick fix for this issue, please give it a test. If you still have any error, please open an issue at https://github.com/fannheyward/coc-postfix/issues

**Comment by hemanth-kotagiri** [comment]
> I can't reproduce, please use latest release branch of coc.nvim.

I've had the same issue with auto-complete along with error detection and it resolved when I updated to the latest branch. Thanks!
